src/util/sample.cpp

```diff
@@ -465,30 +469,39 @@ void SampleUtil::linearCrossfadeBuffersOut(
         const CSAMPLE_GAIN cross_mix = cross_inc * i;
         pDestSrcFadeOut[i * 2] *= (CSAMPLE_GAIN_ONE - cross_mix);
         pDestSrcFadeOut[i * 2] += pSrcFadeIn[i * 2] * cross_mix;
+    }
+    // note: LOOP VECTORIZED. only with "int i"
```

Comment:
What does the `only with "int i"` mean? What would be the alternative to `int i` here that wouldn't work?

Reply:
every other type including SINT

Reply:
Then why don't we introduce a typedef with a single, comprehensible comment instead duplicating it again and again?

Reply:
with a name like `VectorizedLoopIndex` to make it explicit.

Reply:
I think it would help to explicitly document in the comment that SINT does not vectorize.

Reply:
Normally SINT should work in all cases. Here we have an exception that does not work for some reasons. 
I am also not sure which compiler version is affected. 
I have no interest to touch and re-test the code. I will Improve the comment.

Reply:
Done.